packages/eui/src/components/progress/progress.stories.tsx

```diff
@@ -40,11 +39,8 @@ export const Determinate: Story = {
   },
 };
 
-export const Indeterminate: Story = {};
-hideStorybookControls(Indeterminate, [
-  'max',
-  'value',
-  'valueText',
-  'label',
-  'labelProps',
-]);
+export const Indeterminate: Story = {
+  parameters: {
+    controls: { exclude: ['max', 'value', 'valueText', 'label', 'labelProps'] },
```

Comment:
I'm fine using `exclude` as well, but should we use `include` here based on the motivation to be future-proof about having to add new props to exclude them?

```suggestion
controls: { include: ['color', 'position', 'size', 'aria-label']}
```

Reply:
This was a case by case basis - basically the way I was thinking through this was, "if we add another possible prop to progress bars that apply to both indeterminate and determinate, e.g. a second color, I **would** want that prop to show up here, so it makes sense to use `exclude` here since the use case is avoiding only props that apply to determinate progress bars.

That being said I know it's kind of a weird one-off, so I'm totally good with this either way!